core/invoker/src/main/resources/application.conf

```diff
@@ -19,4 +19,12 @@ whisk {
     pause: 10 seconds
     resume: 10 seconds
   }
+
+  # args for 'docker run' to use
+  container-factory.container-args {
```

Comment:
This is great I can add the container root path from #3196 to this struct instead of where I put it.

Reply:
There is a subtle difference here in that the container-args config is not something that is specific to _docker cli_, rather it will apply to non-docker ContainerFactory impls (mesos/kube), while the container root path is very specific to docker cli. 
So I would keep the configs separated to:
- `whisk.docker`
- `whisk.container-factory`